src/SideNav.tsx

```diff
@@ -2,13 +2,20 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import styled, {css} from 'styled-components'
 import classnames from 'classnames'
-import {COMMON, get} from './constants'
-import theme from './theme'
+import {COMMON, get, SystemCommonProps, SystemTypographyProps} from './constants'
 import Link from './Link'
 import BorderBox from './BorderBox'
-import sx from './sx'
+import {ComponentProps} from './utils/types'
+import theme from './theme'
+import sx, {SxProp} from './sx'
+import * as History from 'history'
+
+type StyledSideNavBaseProps = {
```

Comment:
We don't need to use `Styled` here because it's not being passed to a styled-component:
```suggestion
type SideNavBaseProps = {
```